wal/src/virtual_dom/vnode.rs

```diff
@@ -2,16 +2,21 @@ use web_sys::Node;
 
 use super::{VComponent, VElement, VList, VText};
 
+/// VNode is enum representing node in virtual DOM tree. Provides a wrapper over different types of nodes and concise and convinient API for vdom manipulation.
```

Comment:
VNode is enum representing node in virtual DOM tree. It provides a wrapper over different types of nodes along with concise and convinient API for vdom manipulation.